server/src/test/java/org/spine3/server/command/CommandBusShould.java

```diff
@@ -83,26 +89,31 @@
     private TestResponseObserver responseObserver;
 
     @Before
-    public void setUp() {
+    public void setUpTest() {
```

Comment:
Why have the `Test` suffix? It's common to name these methods as `setUp` and `tearDown`. It is assumed that methods annotated `@Before` and `@After` are configuration and clean-up for each test method.